### ISSUE TITLE
loading and saving the synapses?

### ISSUE THREAD

**Issue by jingpengw** [seung-lab/cloud-volume#547]
I am wondering does cloud-volume support point or arrow like synapse representation like this [link](https://h01-dot-neuroglancer-demo.appspot.com/#!gs://h01-release/assets/neuroglancer_states/20210601/synapse_annotations.json)?

**Comment by william-silversmith** [comment]
Hi Jingpeng,

Not yet. I'm hoping to add support for annotations later this year though.

**Comment by jingpengw** [comment]
sounds great! do you know how Google handles it?
Maybe I can work on it.

**Comment by william-silversmith** [comment]
Yes, there's a specification on the Neuroglancer github I can find it for
you in a few hours.

Will

**Comment by william-silversmith** [comment]
Hi Jingpeng,

Here's the specification. If you would have time to build part of this, it would be very helpful! 

https://github.com/google/neuroglancer/blob/master/src/neuroglancer/datasource/precomputed/annotations.md
